src/controllers/subChannel.controller.js

```diff
@@ -88,7 +88,7 @@ exports.deleteSubChannel = async (req, res) => {
 
 exports.getSubChannel = async (req, res) => {
     try {
-        const ChannelId = req.channel._id;
+        const ChannelId = req.params.id
```

Comment:
I think channel is present in the req or req.body added in the middleware. Check the middleware for reference. Donot use param